getflywheel/local-addon-image-optimizer
# Copy Ben's Webpack 4 to 5 upgrade

* Copies [Ben's Webpack 4 to 5 upgrade](https://github.com/getflywheel/local-addon-instant-reload/pull/21)
* Before, `yarn` failed with:

```sh
gtetflywheel/local-addon-image-optimizer/node_modules/loader-runner/lib/LoaderRunner.js:133
		if(isError) throw e;
		            ^

Error: error:0308010C:digital envelope routines::unsupported

```
* This is because NPM packages were outdated.

### Steps to test
1. Start Local with its `master` branch
2. In this repo: `yarn build && npm pack`
3. You'll see a `.tar.gz` file in this repo's root
4. Go to a site with at least 1 large image on it
5. Tools > Image Optimizer > Scan for Images > Optimize Images:

<img width="1265" alt="Screenshot 2023-09-26 at 4 04 23 PM" src="https://github.com/getflywheel/local-addon-image-optimizer/assets/4063887/dbc15319-f50a-4c51-92c7-3a3121da37e7">

6. Expected: 
![optimized-images](https://github.com/getflywheel/local-addon-image-optimizer/assets/4063887/9ade03a7-0e94-47d7-938b-285810ebd687)
